### DISCUSSION TITLE
Extra Blocks

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4348] [Ideas] [answered]
How about adding all of the blocks that snap! has, that scratch DOESN'T have!
https://snap.berkeley.edu/snap

**Comment by Samq64** [comment, marked as answer]
Please stop making suggestions for things that are impossible to implement as a browser extension.

**Comment by LilSefGaming** [comment]
I am not that creative sorry
